### PR TITLE
♿(templates) fix organization glimpse

### PR DESCRIPTION
**Pull request by sandroscosta**
## Purpose
While validating our richie site, I came across an issue with the title on the organization glimpse cards. The link `title` property has the same value as the card span element. That might cause some repetition issues with screen readers. To avoid this, the title property should correctly enunciate that this is a link to the organization page. This way we get better context for our a11y users.

## Proposal
- Add a small fix to the title property the organization glimpse
- The value of title should be different from the organization card

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Added
 
+- Fix organization glimpse card title for accessibility purposes
```
Comment (jbpenrath) [inline CHANGELOG.md:13]: Can you move this entry into the `Fix` section please.

**Review by jbpenrath — APPROVED** [review]
Just one nitpick otherwise LGTM ✅

File: CHANGELOG.md (hunk 1 above)
Comment (sandroscosta) [reply CHANGELOG.md:13]: 'course I can.

**Comment by sandroscosta** [comment]
@jbpenrath apparently something went wrong with the CI, probably due to the rebase.

**Comment by jbpenrath** [comment]
> @jbpenrath apparently something went wrong with the CI, probably due to the rebase.

There are 4 tests which are failing and they looks to be related to your changes.

**Comment by sandroscosta** [comment]
> here are 4 tests which are failing and they looks to be related to your changes.

@jbpenrath I'm clearly not very smart. I just saw it. I'll get right on it. Sorry for wasting your time.

**Pull request merged**